pkg/policyengine/policytypes/connection_request.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 The ClusterLink Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package policytypes
+
+type Direction int
+
+const (
+	Incoming Direction = iota
+	Outgoing
+)
+
+type ConnectionRequest struct {
+	SrcPeer string
+	DstPeer string // ignored on an outgoing connection request
```

Comment:
nit: unclear then when the destination peer is used if ignored on the outgoing request...

Perhaps a more detailed comment (per each exported field) would help?

Reply:
Indeed, `DstPeer` wasn't used, and was removed.
`SrcPeer` is now encoded in the source-workload attributes, so removed it too.